src/openms/source/FORMAT/ChromeleonFile.cpp

```diff
@@ -131,4 +131,9 @@ namespace OpenMS
     ifs.close();
     experiment.addChromatogram(chromatogram);
   }
+
+  double ChromeleonFile::parseDouble(String number) const
```

Comment:
could you rename that function to `parseStringWithComma` or something along those lines?
Because from the signature the reader wonders why you are not simply using the vanilla String::toDouble()

Reply:
I'm going with `removeCommasAndParseDouble()`